shared-bindings/analogio/__init__.c

```diff
@@ -32,6 +32,7 @@
 #include "shared-bindings/microcontroller/Pin.h"
 #include "shared-bindings/analogio/__init__.h"
 #include "shared-bindings/analogio/AnalogIn.h"
+#include "shared-bindings/analogio/AnalogFastIn.h"
```

Comment:
You will have to have the #ifdef around this line as well as the AnalogFastIn line below.